test/render_jsonapi_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+class RenderTest < ActionController::TestCase
+  include Roar::Rails::TestCase
+
+  class SingersController < ActionController::Base
+
+    module SingerRepresenter
+      include Roar::JSON::JSONAPI
+      # type :musicians
+      property :name, :as => :title
+    end
+
+    module SingersRepresenter
+      include Representable::JSON::Collection
+      self.represenation_wrap = :musicians
+      items :extend => PersonRepresenter
+    end
+
+    include Roar::Rails::ControllerAdditions
+    include Roar::Rails::ControllerAdditions::Render
+
+    # NOTE: using :json_api will break
+    represents :json, :entity => SingerRepresenter, :collection => SingersRepresenter
```

Comment:
Ok, for further contributions, keep in mind that you don't have to make tests work with hacks. For the author, it's even better if you submit _breaking_ tests so they can see what's missing. It's actually my favourite PR style when people simply add a broken test.

Thanks, I'm pushing soon!